libs/cart/src/facades/cart/cart.facade.ts

```diff
@@ -4,19 +4,61 @@ import { Action, Store, select } from '@ngrx/store';
 
 import { DaffStoreFacade } from '@daffodil/core';
 import { DaffCart } from '../../models/cart';
-import * as fromCart from '../../reducers';
+import { DaffCartReducersState } from '../../reducers';
+import {
+  selectCartLoading,
+  selectCartValue,
+  selectCartErrors,
+  selectCartId,
+  selectCartSubtotal,
+  selectCartGrandTotal,
+  selectCartCoupons,
+  selectCartItems,
+  selectCartBillingAddress,
+  selectCartShippingAddress,
+  selectCartPayment,
+  selectCartTotals,
+  selectCartShippingInformation,
+  selectCartAvailableShippingMethods,
+  selectCartAvailablePaymentMethods
+} from '../../selectors';
 
 @Injectable({
   providedIn: 'root'
 })
 export class DaffCartFacade implements DaffStoreFacade<Action> {
-
   loading$: Observable<boolean>;
   cart$: Observable<DaffCart>;
-  
-  constructor(private store: Store<fromCart.State>) {
-    this.loading$ = this.store.pipe(select(fromCart.selectCartLoadingState));
-    this.cart$ = this.store.pipe(select(fromCart.selectCartValueState));
+  errors$: Observable<string[]>;
+  cartId$: Observable<DaffCart['id']>;
```

Comment:
you can remove all of the "cart" prefixes. Since this is coming from the cart facade, it should be assumed that all these properties pertain to cart.

Reply:
Changed in [`fdf6e8b`](https://github.com/graycoreio/daffodil/pull/681/commits/fdf6e8b2be945c8ad297e46ca13638b50cfc2191).